src/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -162,4 +162,7 @@
   <data name="ProjectsHeader" xml:space="preserve">
     <value>Project(s)</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="NoSolutionFolders" xml:space="preserve">
+    <value>Do not create solution folders. Create project at root of solution.</value>
```

Comment:
@KathleenDollard to review this `dotnet sln add` option string.

My suggestion would be:

```suggestion
    <value>Add project to the root of the solution without creating any solution folders.</value>
```